src/test/java/com/spirnt/mission/discodeit/integrationTests/ChannelTest.java

```diff
@@ -0,0 +1,161 @@
+package com.spirnt.mission.discodeit.integrationTests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.spirnt.mission.discodeit.dto.ErrorResponse;
+import com.spirnt.mission.discodeit.dto.channel.ChannelDto;
+import com.spirnt.mission.discodeit.dto.channel.PrivateChannelCreateRequest;
+import com.spirnt.mission.discodeit.dto.channel.PublicChannelCreateRequest;
+import com.spirnt.mission.discodeit.dto.channel.PublicChannelUpdateRequest;
+import com.spirnt.mission.discodeit.entity.ChannelType;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Transactional;
+
+@ActiveProfiles("test")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class ChannelTest {
+
+  @Autowired
+  private TestRestTemplate restTemplate;
+
+  @Test
+  @Transactional
+  @DisplayName("공개 채널 생성 테스트 - 성공")
+  void testCreatePublicChannelSuccess() {
+    PublicChannelCreateRequest publicChannelCreateRequest = new PublicChannelCreateRequest("ch 1",
+        "this is ch 1");
+    ResponseEntity<ChannelDto> response = restTemplate.postForEntity("/api/channels/public",
+        publicChannelCreateRequest, ChannelDto.class);
+    assertEquals(response.getStatusCode(), HttpStatus.CREATED);
```

Comment:
assertEquals는 첫번째 파라미터가 expected라서 순서가 뒤바뀌어야할 것 같아요.  
```suggestion
    assertEquals(HttpStatus.CREATED, response.getStatusCode());
```

![스크린샷 2025-04-09 오후 7 50 47](https://github.com/user-attachments/assets/dbb909a7-bcb0-41d3-a55a-d7f288baec19)